StitchCore-iOS/StitchCore_iOSTests/Core/Auth/Providers/UserAPIKey/UserAPIKeyAuthProviderTests.swift

```diff
@@ -0,0 +1,201 @@
+import Foundation
+import XCTest
+import StitchCore
+import StitchCore_iOS
+
+class UserAPIKeyAuthProviderTests: StitchIntegrationTestCase {
```

Comment:
[suggestion] You may want some failure test cases here.
[request] Can you file a ticket for the unit tests for this provider in core?

Reply:
Ticket is STITCH-1537